packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -15,12 +15,19 @@
             <span data-ng-bind=":: $ctrl.vps.vcore"></span>
             <p
```

Comment:
Any reason to keep the `p` tag ?

Reply:
Yes, `span + p > a`, to avoid `a.d-block`